### DISCUSSION TITLE
WAF功能支持对单网站设置IP黑白名单

### DISCUSSION THREAD

**Discussion by auth007** [1Panel-dev/1Panel#6269] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 请描述您的需求或者改进建议

当前只能设置全局黑白名单，如果网站比较多可能出现冲突问题。建议支持对单个网站设置IP黑白名单

**Comment by zhengkunwang223** [comment]
![image](https://github.com/user-attachments/assets/eb329443-49d6-4c56-963a-8bc8180a211e)
专业版支持

**Reply by auth007** [reply]
只能单独IP。IP组不支持。

**Reply by zhengkunwang223** [reply]
> 只能单独IP。IP组不支持。

V2 会优化一下

**Comment by Jas0nxlee** [comment]
然而现在的版本并没有这个功能。
<img width="1438" alt="image" src="https://github.com/user-attachments/assets/3335e7be-3e0c-4bcc-8496-f00b46e34216" />

**Reply by zhengkunwang223** [reply]
> 然而现在的版本并没有这个功能。 <img alt="image" width="1438" src="https://private-user-images.githubusercontent.com/33084918/434667374-3335e7be-3e0c-4bcc-8496-f00b46e34216.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ4NzI0OTgsIm5iZiI6MTc0NDg3MjE5OCwicGF0aCI6Ii8zMzA4NDkxOC80MzQ2NjczNzQtMzMzNWU3YmUtM2UwYy00YmNjLTg0OTYtZjAwYjQ2ZTM0MjE2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDE3VDA2NDMxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI2NTZkNmYxN2Y0MjE2YTY2ZDQ3NmNjODIwZGVjNzBiZGJkM2JhN2I5MDk3NzYxNzkxZGI1NWM5ZGJmZDRhNWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.LLKhbd24VfoqyQUlzAyes6KDZmUEMNwaUi_sS09s7qA">

专业版支持  购买链接 https://www.lxware.cn?code=vip88888
